Modix/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -93,7 +95,7 @@ public static IServiceCollection AddModix(this IServiceCollection services)
 
                     service.AddTypeReader<IEmote>(new EmoteTypeReader());
                     service.AddTypeReader<DiscordUserEntity>(new UserEntityTypeReader());
-                    service.AddTypeReader<DiscordUserMessage>(new DiscordUserMessageTypeReader());
+                    service.AddTypeReader<IMessage>(new MessageTypeReader(), true);
```

Comment:
This effectively overrides the existing `IMessage` reader?

Reply:
Correct. It didn't look like we were using `IMessage` in any command parameters, and the places we could change to use `IMessage` (e.g. `QuoteModule`) would benefit from the extra capabilities anyway